cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ConfigRegistrationRequest.java

```diff
@@ -19,16 +19,20 @@
     @JsonProperty
     private List<ClusterServiceConfig> services;
 
+    @JsonProperty
+    private List<String> certificates;
+
     @JsonCreator
-    public ConfigRegistrationRequest(String clusterCrn, List<String> aliases, List<ClusterServiceConfig> services) {
+    public ConfigRegistrationRequest(String clusterCrn, List<String> aliases, List<ClusterServiceConfig> services, List<String> certificates) {
```

Comment:
didn't we also need to add the `tlsStrictCheck` field?

Reply:
Good catch, it looks like they are defaulting that field as false for now, but will likely change in the future, I'll add the field in.